### PR TITLE
store/tikv: use unistore instead of mocktikv in unit test

### PR DESCRIPTION
**Pull request by coocood**
### What problem does this PR solve?

For removing obsolete mocktikv code.

### What is changed and how it works?

Use unistore instead of mocktikv in unit test of store/tikv package.

### Check List 

Tests 

- Unit test

### Release note 

- no release note

### CODE REVIEW

**Comment by tiancaiamao** [comment]
LGTM

File: store/tikv/scan.go [hunk 1]
```diff
@@ -238,7 +238,7 @@ func (s *Scanner) getData(bo *Backoffer) error {
 		kvPairs := cmdScanResp.Pairs
 		// Check if kvPair contains error, it should be a Lock.
 		for _, pair := range kvPairs {
-			if keyErr := pair.GetError(); keyErr != nil {
+			if keyErr := pair.GetError(); keyErr != nil && len(pair.Key) == 0 {
```
Comment (lysu) [inline store/tikv/scan.go:241]: why we should add this check ?- -

Comment (coocood) [reply store/tikv/scan.go:241]: Because of a bug in unistore repo, this change can be removed after unistore is updated.

**Pull request merged**